packages/replay-next/components/inspector/ScopesInspector.tsx

```diff
@@ -23,23 +25,49 @@ export default function ScopesInspector({
 }) {
   return (
     <div className={styles.ScopesInspector} data-test-name="ScopesInspector">
-      <Suspense fallback={<Loader />}>
-        <Expandable
-          className={styles.Expandable}
-          children={protocolValues.map((protocolValue, index) => (
-            <Inspector
-              context="default"
-              key={index}
-              path={addPathSegment(path, protocolValue.name)}
-              pauseId={pauseId}
-              protocolValue={protocolValue}
-            />
-          ))}
-          defaultOpen={expandByDefault}
-          header={name}
-          persistenceKey={path}
-        />
-      </Suspense>
+      <Expandable
+        className={styles.Expandable}
+        defaultOpen={expandByDefault}
+        header={name}
+        persistenceKey={path}
+      >
+        <Suspense fallback="Loading...">
+          <ScopesInspectorChildren path={path} pauseId={pauseId} protocolValues={protocolValues} />
+        </Suspense>
```

Comment:
Actually after some more experimentation I found that this isn't necessary to fix the performance. But previously I had placed the prefetching in `ScopesInspector` and in that case moving the `Suspense` boundary inside the `Expandable` fixed the performance (which I found super baffling).
So while this isn't necessary for performance I want to keep it because the `Expandable` shouldn't disappear while its children are loading.

Reply:
> I want to keep it because the `Expandable` shouldn't disappear while its children are loading.

Yeah, I agree.